salesagility/SuiteCRM
# Fix #10433 - Email Address ID being double quoted

## Description




Fixes issue of the email address ID being double quoted which causes a mysql in the log


## Motivation and Context


Stops MySQL error in log

## How To Test This


Go to user in Admin, edit email address and then check log for error.
This fix stops the error.

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)

### Final checklist


- [x ] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.
